### PR TITLE
added videojs integration example

### PR DESCRIPTION
**Pull request by prateekrastogi**

### CODE REVIEW

File: examples/with-videojs/components/Player.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react'
+import videojs from 'video.js'
+import 'videojs-youtube'
+import '../node_modules/video.js/dist/video-js.css'
```
Comment (timneutkens) [inline examples/with-videojs/components/Player.js]: You can actually use: `video.js/dist/video-js.css` from 6.0.0-canary.2

Comment (prateekrastogi) [reply examples/with-videojs/components/Player.js]: I opened and closed an issue on next-plugins repo. Somehow, it is not working properly with next-css plugin and throwing error:

 DONE  Compiled successfully in 4459ms                                                                                                                                       19:11:21
C:\Code\Starters\next.js\examples\with-videojs\node_modules\video.js\dist\video-js.css:1
(function (exports, require, module, __filename, __dirname) { .video-js .vjs-big-play-button .vjs-icon-placeholder:before, .vjs-button > .vjs-icon-placeholder:before, .video-js .vjs-modal-dialog, .vjs-modal-dialog .vjs-modal-dialog-content {
                                                              ^

SyntaxError: Unexpected token .
    at createScript (vm.js:80:10)
    at Object.runInThisContext (vm.js:139:10)
    at Module._compile (module.js:607:28)
    at Object.Module._extensions..js (module.js:654:10)
    at Module.load (module.js:556:32)
    at tryModuleLoad (module.js:499:12)
    at Function.Module._load (module.js:491:3)
    at Module.require (module.js:587:17)
    at require (internal/module.js:11:18)
    at Object.video.js/dist/video-js.css (C:\Code\Starters\next.js\examples\with-videojs\.next\dist\bundles\pages\webpack:\external "video.js\dist\video-js.css"?ab25:1:1)
    at __webpack_require__ (C:\Code\Starters\next.js\examples\with-videojs\.next\dist\bundles\pages\webpack:\webpack\bootstrap 055c3930bca32ef0b13e?e39d:21:1)
    at Object../components/Player.js (C:\Code\Starters\next.js\examples\with-videojs\.next\dist\bundles\pages\index.js:85:85)
    at __webpack_require__ (C:\Code\Starters\next.js\examples\with-videojs\.next\dist\bundles\pages\webpack:\webpack\bootstrap 055c3930bca32ef0b13e?e39d:21:1)
    at Object../pages/index.js (C:\Code\Starters\next.js\examples\with-videojs\.next\dist\bundles\pages\index.js:184:77)
    at __webpack_require__ (C:\Code\Starters\next.js\examples\with-videojs\.next\dist\bundles\pages\webpack:\webpack\bootstrap 055c3930bca32ef0b13e?e39d:21:1)
    at Object.2 (C:\Code\Starters\next.js\examples\with-videojs\.next\dist\bundles\pages\index.js:266:18)

Comment (timneutkens) [reply examples/with-videojs/components/Player.js]: Again:

You can actually use: video.js/dist/video-js.css ***from 6.0.0-canary.2***, there was a bug that has been fixed in that version 👍

***from 6.0.0-canary.2*** 😄

Comment (prateekrastogi) [reply examples/with-videojs/components/Player.js]: Thanks, I will create a new pull request using that version.

**Pull request closed without merging**